### PR TITLE
Rename "graphs" to "networks" in UI, and make network list primary

### LINKED ISSUE #278: Rename "Graphs" to Networks, and update order
image

This should be "Networks" (we don't want to use the work Graph ever); and since Networks are the key thing, we want to put this to the left.

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #278.

This PR doesn't refactor the use of the word "graph" from the codebase itself, just where it's visible within the UI (see #291).

Reviewers, please point out if I missed any of these.

### CODE REVIEW

File: client/src/views/WorkspaceDetail.vue [hunk 1]
```diff
@@ -80,19 +80,21 @@
             text
           >
             <item-panel
-              ref="tablePanel"
-              title="Tables"
-              :items="tables"
+              ref="graphPanel"
+              title="Networks"
+              :items="graphs"
```
Comment (JackWilb) [inline client/src/views/WorkspaceDetail.vue]: Why did you change the order of these? Better UX?

Comment (waxlamp) [reply client/src/views/WorkspaceDetail.vue]: Alex requested this change to make "networks" front-and-center. (That's what I meant by "make network list primary" in the issue title, but that was probably unclear.)

Comment (JackWilb) [reply client/src/views/WorkspaceDetail.vue]: Gotcha!
